README.md

```diff
@@ -63,10 +64,11 @@ npm run sass-ms
 
 ### Build lf-documentation in watch mode
 
-This allows you to see changes you make to the ui components quickly.
+This allows you to see changes you make to the ui components quickly. You can run the two commands in different windows so that you can see any changes in the browser, even those that are shared across components.
```

Comment:
we can also add this to the readme in lf-documentation. (or should we delete readme at secondary entrypoints?